src/main/java/com/teamabnormals/abnormals_core/core/AbnormalsCore.java

```diff
@@ -84,12 +86,17 @@ public AbnormalsCore() {
 			}
 		});
 
-		DistExecutor.runWhenOn(Dist.CLIENT, () -> () -> {
-			((IReloadableResourceManager) Minecraft.getInstance().getResourceManager()).addReloadListener(ENDIMATION_DATA_MANAGER);
-			modEventBus.addListener(this::clientSetup);
-		});
+		DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> () -> modEventBus.addListener(EventPriority.NORMAL, false, ColorHandlerEvent.Block.class, event ->
```

Comment:
Some code in this segment isn't consistent with the formatting of this repository, such as new-line brackets.